tests/integration/Tribe/Admin/OrderReportTest.php

```diff
@@ -77,8 +77,8 @@ function (): array {
 				$ticket_id = $this->create_tc_ticket( $event_id, 10 );
 
 				$this->set_fn_return( 'current_time', '2020-02-22 22:22:22' );
-
 				$order_a = $this->create_order( [ $ticket_id => 2 ], [ 'purchaser_email' => 'purchaser@test.com' ] );
+				$this->set_fn_return( 'current_time', '2020-02-23 22:22:22' );
```

Comment:
I understand why this might be required in concept, either:
1. Remove this and use another method to update the Order date (e.g. targeted `update_post_meta` or `wp_update_post`) after you've inserted the order.
2. Add a comment to explain why you're doing this.

Of the two, I would prefer the first option: altering time can very easily have unforeseen consequences and implications that are not immediate. If what you're trying to do is simulate an order placed in the past, just alter the order data.

Reply:
I actually tried the 1st method that you proposed. But the thing is once a post is updated the created/updated date is also updated. that's a loop. Doesn't matter if you are updating the created/updated date or not.

Reply:
`update_post_meta` won't help as the date is coming from the post table data.